warcit/warcit.py

```diff
@@ -153,13 +216,32 @@ def __init__(self, url_prefix, inputs,
                 self.mime_overrides[p[0]] = p[1]
 
         self.use_magic = use_magic
-        self.magic = None
+        self.no_xhtml = no_xhtml
 
         self.charset = charset
 
+        self.include = None
+        if include:
+            self.include = [x.lower() for x in include.split(',')]
+        self.exclude = None
+        if exclude:
+            self.exclude = [x.lower() for x in exclude.split(',')]
+
+        self.use_tika = self.use_magic == 'tika' or self.charset == 'tika'
```

Comment:
This seems confusing.. seems like tika can be enabled with either `--use-magic tika` or `--charset tika`?
If so, maybe can remove the `--use-magic` param or have it remain true/false only, overridable by charset.
It seems like there's two different params to do the same thing?

Reply:
Charset and mime type detection are separated. You can use tika for detecting mime types  and/or use it for charset detection. tika is much better at detecting charsets, than cchardet, but sometimes not so practical for detecting mime-types (too specific). This needs documentation. The main thing is that the defaults are the same as before and good for most cases.

Reply:
Got it, i think i was confused by the docs :) Perhaps the flag could also be `--mime` as well as `--use-magic`